src/Persistence/MassTransit.RedisIntegration/RedisSagaRepository.cs

```diff
@@ -126,14 +126,30 @@ public void Probe(ProbeContext context)
 
         async Task UpdateRedisSaga(TSaga instance)
         {
-            ITypedDatabase<TSaga> sagas = _redisDbFactory().As<TSaga>();
+            RedisKey key = $"saga_lock_{instance.CorrelationId}";
+            RedisValue token = Environment.MachineName + Guid.NewGuid();
+            var db = _redisDbFactory();
```

Comment:
The dotSettings file says that you need to explicitly specify types, could you please fix this `var`.